dotCMS/src/main/java/com/dotmarketing/business/PermissionBitFactoryImpl.java

```diff
@@ -133,41 +139,41 @@
 	 * 2. Permission type
 	 * 3. asset id
 	 */
-	private final String updatePermissionReferenceByAssetIdSQL = "update permission_reference set reference_id = ? where permission_type = ? and asset_id = ?";
+	private static final String updatePermissionReferenceByAssetIdSQL = "update permission_reference set reference_id = ? where permission_type = ? and asset_id = ?";
 
 	/*
 	 * Select permission references based on how are referencing the type of reference
 	 * Parameters
 	 * 1. Reference id
 	 * 2. Permission type
 	 */
-	private final String selectPermissionReferenceSQL = "select asset_id from permission_reference where reference_id = ? and permission_type = ?";
+	private static final String selectPermissionReferenceSQL = "select asset_id from permission_reference where reference_id = ? and permission_type = ?";
 
 	/*
 	 * To remove a permission reference of an specific asset or referencing an asset
 	 *
 	 */
-	private final String deletePermissionReferenceSQL = "delete from permission_reference where asset_id = ? or reference_id = ?";
+	private static final String deletePermissionReferenceSQL = "delete from permission_reference where asset_id = ? or reference_id = ?";
 
 	/*
 	 * To remove all permission references
 	 *
 	 */
-	private final String deleteAllPermissionReferencesSQL = "delete from permission_reference";
+	private static final String deleteAllPermissionReferencesSQL = "delete from permission_reference";
 
 
 	/*
 	 * To remove a permission reference of an specific asset
 	 *
 	 */
-	private final String deletePermissionableReferenceSQL = "delete from permission_reference where asset_id = ?";
+	private static final String deletePermissionableReferenceSQL = "delete from permission_reference where asset_id = ?";
 
 	/*
 	 * To load template identifiers that are children of a host
 	 * Parameters
 	 * 1. The id of the host
 	 */
-	private final String selectChildrenTemplateSQL =
+	private static final String selectChildrenTemplateSQL =
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Rename this constant name to match the regular expression '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00115)